etc/check_license.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+year=$(date +"%Y")
+copyright=$"// Copyright (C) MongoDB, Inc. $year-present.
+//
+// Licensed under the Apache License, Version 2.0 (the \"License\"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+"
+
+add_copyright() {
+    file=$1
+    check_only=$2
+    verbose=$3
```

Comment:
```suggestion
```
This is set by the getopts while loop, like "add".

Reply:
Good catch! I realize now that `$check_only` also needs to be removed and that `$add` is only working because it is accessible globally.